### ISSUE TITLE
File name <> Module name inconsistent

### ISSUE THREAD

**Issue by tomverbeure** [SpinalHDL/VexRiscv#38]
File name: GenSmallAndPerformant.scala
Module name: GenSmallAndProductive

Same thing for: GenSmallAndProductiveWithICache

Which is the right name: *performant or *productive?

Once I know, I'll submit a patch.

Tom

**Comment by tomverbeure** [comment]
Also: the GenFullNoMmuNoCache.scala and GenNoCacheNoMmuMaxPerf.scala files are identical.

**Comment by Dolu1990** [comment]
Right, some of the naming is twisted, including with the one reported and not 100% coherent with the one reported in the readme. I will have to fix it.
For  GenFullNoMmuNoCache.scala and GenNoCacheNoMmuMaxPerf.scala, the difference is in the branch plugin. DYNAMIC_TARGET vs STATIC prediction, isn't it ?
